styleguide/derek/_partials/grid/widget/ias.ejs

```diff
@@ -2,24 +2,13 @@
 <% if(!desc) { var desc = "This description cannot be longer than 75 characters including all spaces."; } %>
 <!-- Infrastructure and Services Pattern -->
 
-<a href="http://www.google.com">
-  <div class="row">
-    <div class="col-sm-4 col-xs-3 center-block center">
-      <img src="http://broadcast.rackspace.com/r/sites/test/text/images/70px.png" alt="">
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-md-12">
-      <h3 class="center">Name with only 30 characters</h3>
-    </div>
-    <div class="col-xs-3 center-block">
-      <hr>
-    </div>
+<a class="iasWidget iasWidget-hover" href="http://www.google.com">
+  <img class="iasWidget-img" src="https://placeholdit.imgix.net/~text?txtsize=30&txt=%5Bplaceholder%5D&w=200&h=150">
+  <div class="iasWidget-row">
     <div class="col-md-12">
```

Comment:
Change this to an `iasWidget-col-full` class and then add mixin to emulate behavior.